lib/client/ipam_test.go

```diff
@@ -170,8 +180,86 @@ var _ = Describe("IPAM tests", func() {
 		Entry("Assign 1 IPv4 address with AssignIP then try to release 2 IPs (release a second one)", net.ParseIP("192.168.1.1"), false, []string{"192.168.1.0/24", "fd80:24e2:f998:72d6::/120"}, net.IP{}, 0, []cnet.IP{cnet.IP{net.ParseIP("192.168.1.1")}}, nil),
 	)
 
+	DescribeTable("ClaimAffinity: claim IPNet vs actual number of blocks claimed",
+		func(inNet string, host string, cleanEnv bool, pool []string, assignIP net.IP, expClaimedIPs, expFailedIPs int, expError error) {
```

Comment:
In general, I think once you've got this many args to a function, it is better to wrap them up in a struct so that they're easier to parse.

This will make reading the individual test cases easier as well.